src/vs/base/browser/ui/sash/sash.ts

```diff
@@ -130,6 +139,7 @@ export class Sash extends Disposable {
 	constructor(container: HTMLElement, layoutProvider: ISashLayoutProvider, options: ISashOptions = {}) {
 		super();
 
+		Sash.size = isIPad ? maxSize : minSize;
```

Comment:
Let's move this whole `isPad` dance over to our contribution: the default value of our configuration setting would be based on the `isIPad` condition. That way we can also move the `minSize` and `maxSize` constants directly into the configuration contribution.

Reply:
I don't think we should set the default value based on the `isIPad` condition since this setting could be synched to other machines. I think that if `isIPad` is true, we should just use `20` for the size of the sash independent what the setting is set to.